setup.py

```diff
@@ -50,9 +50,11 @@
 		'viewer': ['cherrypy>=3.2.3', 'jinja2'],
 		'slack': ['slackclient', 'slacker'],
 		'irc': ['irc>=15.0,<16dev'],
+		'newrelic': ['yg.newrelic>=1.13,<2dev'],
 	},
 	setup_requires=[
 		'setuptools_scm>=1.15.0',
+		'wheel>=0.25.0',
```

Comment:
I'm not sure I understand why pmxbot would need to be built with a specific wheel version. Also, I've been trying to limit the number of dependencies in `setup_requires` because (a) they get installed at any invocation of setup.py even when they're not needed and (b) they use `easy_install` which I'm trying to avoid.